assets/src/design-system/components/input/index.js

```diff
@@ -62,14 +77,22 @@ const StyledInput = styled.input(
 
     :active {
       border-color: ${theme.colors.border.defaultActive};
+
+      & ~ div svg {
```

Comment:
You can actually be more specific in the selector here by doing:

```css
      & ~ ${IconContainer} svg {
```

And similarly for `~ p` a little further below.

S-C will [replace this with the generated class name](https://styled-components.com/docs/advanced#referring-to-other-components) - pretty nifty.